### PR TITLE
Only bind jmxremote management interface to 127.0.0.1 for pricenodes

### PR DESCRIPTION
**Pull request by wiz**

### CODE REVIEW

File: monitor/install_collectd_debian.sh [hunk 1]
```diff
@@ -32,8 +32,8 @@ echo "[*] Preparing Bisq init script for monitoring"
...
-        sudo -H -i -u "${ROOT_USER}" sed -i -e '/JAVA_OPTS/ s/"$/ -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.local.only=true -Dcom.sun.management.jmxremote.port=6969 -Dcom.sun.management.jmxremote.rmi.port=6969 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"/' "${file}"
+        sudo -H -i -u "${ROOT_USER}" sed -i -e 's/-Dcom.sun.management.jmxremote //g' -e 's/-Dcom.sun.management.jmxremote.local.only=true//g' -e 's/ -Dcom.sun.management.jmxremote.host=127.0.0.1//g' -e '/ -Dcom.sun.management.jmxremote.host=127.0.0.1//g' -e 's/ -Dcom.sun.management.jmxremote.port=6969//g' -e 's/ -Dcom.sun.management.jmxremote.rmi.port=6969//g' -e 's/ -Dcom.sun.management.jmxremote.ssl=false//g' -e 's/ -Dcom.sun.management.jmxremote.authenticate=false//g' "${file}"
```
Comment (devinbileck) [inline monitor/install_collectd_debian.sh]: Is this necessary?
As immediately before it we are substituting `jmxremote.host=127.0.0.1`

Comment (wiz) [reply monitor/install_collectd_debian.sh]: oh wow yes true, seems florian is removing the setting here but not adding it in section below

**Review by devinbileck — APPROVED** [review]
ACK

**Review by ripcurlx — APPROVED** [review]
utACK

**Pull request merged**